From 6662e6c683c9535fc0222d7bf53989448899be0b Mon Sep 17 00:00:00 2001
From: Wang Jie <dave.wang@rock-chips.com>
Date: Thu, 16 Apr 2020 09:54:50 +0800
Subject: [PATCH] arm64: rockchip_defconfig: enable CONFIG_GS_MXC6655XA.

Because many customers have already started using mxc6655xa, so enable it.

Change-Id: I9aa83a9342b1a7a70da8cdb07ce88c9971d3230e
Signed-off-by: Wang Jie <dave.wang@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index 2b66c68f7b0e..e56b01cde108 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -446,6 +446,7 @@ CONFIG_GS_KXTJ9=y
 CONFIG_GS_LIS3DH=y
 CONFIG_GS_MMA7660=y
 CONFIG_GS_MC3230=y
+CONFIG_GS_MXC6655XA=y
 CONFIG_GS_LSM303D=y
 CONFIG_LSM330_ACC=y
 CONFIG_BMA2XX_ACC=y
-- 
2.35.3

